avogadro/qtplugins/overlayaxes/overlayaxes.cpp

```diff
@@ -274,7 +276,9 @@ void OverlayAxes::processAxis()
     m_scene->rootNode().removeChild(engineNode);
     delete engineNode;
     m_widgetToNode[m_glWidget] = nullptr;
-  } else {
+  }
+
+  if (m_enabled) {
     engineNode = new Rendering::GroupNode(&node);
```

Comment:
@serk12 - on launch, the code crashes here...

Reply:
There is a memory leak with `m_scene` when the `OverlayAxes` constructor calls `processAxes` remove it  (line 246?) and it should work.

Reply:
@serk12 - that removes the crash, but then the user doesn't see the axes when enabled. Maybe running `processAxes` when `setMolecule` is called?

I'm wondering if it isn't worth reverting to a `render` plugin.